app/src/main/res/menu/episodes_apply_action_options.xml

```diff
@@ -9,17 +9,17 @@
         app:showAsAction="always">
         <menu>
             <item android:id="@+id/sort_title_a_z"
-                  android:title="@string/sort_title_a_z"/>
+                android:title="@string/sort_title_a_z"/>
```

Comment:
Please don't reformat the file, unless you really need to change it. We don't have clear guidelines for xml formatting currently, so it should just be kept as it is.

Reply:
Still unrelated changes here :)